addon/components/rdf-input-fields/alert.js

```diff
@@ -2,6 +2,7 @@ import Component from '@glimmer/component';
 
 export default class AlertComponent extends Component {
   get options() {
+    // Not changin this to findFieldOption() See input-field.js
     return this.args.field.options;
   }
```

Comment:
Here are the field.options also used. Just like int the case-number and the heading component. Should we use our new way to get those?

* ext:title
* ext:skin
* ext:level
* ext:prefix
* ext:icon
* ext:closable
* ext:size